nsxt/resource_nsxt_policy_lb_client_ssl_profile.go

```diff
@@ -0,0 +1,210 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+var lBClientSslProfileCipherGroupLabelValues = []string{
+	model.LBClientSslProfile_CIPHER_GROUP_LABEL_HIGH_COMPATIBILITY,
+	model.LBClientSslProfile_CIPHER_GROUP_LABEL_HIGH_SECURITY,
+	model.LBClientSslProfile_CIPHER_GROUP_LABEL_CUSTOM,
+	model.LBClientSslProfile_CIPHER_GROUP_LABEL_BALANCED,
+}
+
+func resourceNsxtPolicyLBClientSslProfile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyLBClientSslProfileCreate,
+		Read:   resourceNsxtPolicyLBClientSslProfileRead,
+		Update: resourceNsxtPolicyLBClientSslProfileUpdate,
+		Delete: resourceNsxtPolicyLBClientSslProfileDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"nsx_id":       getNsxIDSchema(),
+			"path":         getPathSchema(),
+			"display_name": getDisplayNameSchema(),
+			"description":  getDescriptionSchema(),
+			"revision":     getRevisionSchema(),
+			"tag":          getTagsSchema(),
+			"cipher_group_label": {
```

Comment:
Sounds from the description this should be Computed only?

Reply:
The doc says below `A label of cipher group which is mostly consumed by GUI. The server will populate this field when returing the resource`. So one of those seems incorrect - either the doc, or the code

Reply:
I think the doc is incorrect, it's fixed now.

> The server will populate this field when returing the resource.  Ignored on PUT and POST.
These are comments for the Links field in the model package, not sure why it ended up in the cipher_group_label after auto-generating the doc. Thanks for catching this